route/router.go

```diff
@@ -10,9 +10,35 @@ import (
 type Router interface {
 	// Options configures the router with the given options. An error is
 	// returned if validation issues exist.
-	Options(opts ...Option) error
+	Options(...Option) error
 	// Solver receives solve options and returns a Solver interface.
-	Solver(opt store.Options) (store.Solver, error)
+	Solver(store.Options) (store.Solver, error)
+	// Plan returns a variable which holds information
+	// about the current set of unassigned stops and vehicles.
+	// The plan variable can be used to retrieve the values from the
+	// store of a solution.
+	Plan() store.Variable[Plan]
+}
+
+// Plan describes a solution to a Vehicle Routing Problem.
+type Plan struct {
+	Unassigned []Stop    `json:"unassigned"`
+	Vehicles   []Vehicle `json:"vehicles"`
+}
+
+// Vehicle information of a solution to a Vehicle Routing Problem.
+type Vehicle struct {
+	ID            string        `json:"id"`
+	Route         []VehicleStop `json:"route"`
+	RouteDuration int           `json:"route_duration"`
+}
+
+// VehicleStop describes a stop as part of a Vehicle's route of solution
+// to a Vehicle Routing Problem.
+type VehicleStop struct {
+	Stop
+	ETA *time.Time `json:"eta,omitempty"`
+	ETD *time.Time `json:"etd,omitempty"`
```

Comment:
```suggestion
	EstimatedArrival *time.Time `json:"estimated_arrival,omitempty"`
	EstimatedDeparture *time.Time `json:"estimated_departure,omitempty"`
```

I always steer clear from acronyms to avoid people asking "what does xyz stand for?" In addition, `router` already uses the full name so I think it makes sense to stick with it.

Reply:
Yeah, good point. Then let's go with the non-acronym version (ETA,ETD) is the current API. This also simplifies some things on the implementation part.